templates/artist/includes/artist_update_media.html

```diff
@@ -3,7 +3,9 @@
 {% for media_url in update.updatemediaurl_set.all %}
     {% if not forloop.first %}<br />{% endif %}
     {% if embed %}
-        {{ media_url.embed_html|safe }}
+        <div class="videowrapper">
```

Comment:
Arguably this `<div>` should actually be part of the HTML generated from `embed_html()`. Check out `artist/models.py` line 268. That section of code should be changed to look something like:
```python
return u"""
    <div class="videowrapper">
        <iframe width="560" height="315" src="{url}" frameborder="0" allowfullscreen></iframe>
    </div>
""".format(url=url)
```